### ISSUE TITLE
Logging results to test for errors

### ISSUE THREAD

**Issue by dmoonfire** [muesli/beehive#225]
So, this is related to my first few attempts at using BeeHive. I have no clue what was going on and if my task worked. After the run, it said it worked but it didn't post so I figured *something* was wrong. However, I couldn't find a log file to tell me what went wrong and which step I had off.

Integromat (which I'm currently) using has the ability to see the last X runs and information (such as Retrieved RSS,3 new items, picking 1). That is sometimes useful to know if I had a problem with my authentication (my first attempt with Mastodon), a filter (second attempt).

**Comment by muesli** [comment]
You're absolutely right, Beehive is still lacking in that area and I'm aware of that. There is already a logging endpoint for the API in the works, but the admin interface still needs to be updated to make use of that.

Hopefully something for the next release!
